.readthedocs.yml

```diff
@@ -1,6 +1,41 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html
+# for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  builder: dirhtml
+  configuration: docs/conf.py
+  fail_on_warning: true  # https://github.com/pypa/setuptools/issues/2424
+
+# Build documentation with MkDocs
+#mkdocs:
+#  configuration: mkdocs.yml
+#  fail_on_warning: true
+
+# Optionally build your docs in additional formats
+# such as PDF and ePub
+formats:
+- epub
+- pdf
+- htmlzip
+
+submodules:
+  include: all  # []
+  exclude: []
+  recursive: true
+
+build:
+  image: latest
+
+# Optionally set the version of Python and requirements required
+# to build docs
 python:
-  version: 3
-  extra_requirements:
-  - docs
-  pip_install: false
-  requirements: docs/requirements.txt
+  version: 3.8
+  install:
+  - requirements: docs/requirements.txt
+  system_packages: false
```

Comment:
Does v2 require this setting for all projects?

Reply:
Probably not, I copied it from cheroot. I don't remember why I added it but I guess I wanted to be explicit about what's happening.